### PR TITLE
[README] Converted Class Component to Function Component

### PR DESCRIPTION
**Pull request by markokarapandzic**
In the **Usage part** of the README.
I converted the example code from a Class Component to Function Component.
So it will be **up-to-date** and **Hooks** can be used with it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,152 +216,157 @@ import {
...
+} from 'react-native-twilio-video-webrtc';
+
+const CallScreen = (props) => {
+  const [isAudioEnabled, setIsAudioEnabled] = useState(true);
+  const [isVideoEnabled, setIsVideoEnabled] = useState(true);
+  const [status, setStatus] = useState('disconnected');
+  const [participants, setParticipants] = useState(new Map());
+  const [videoTracks, setVideoTracks] = useState(new Map());
+  const [token, setToken] = useState('');
+  let twilioRef = null;
```
Comment (fabiendem) [inline README.md:228]: I think we can use `useRef` here
```js
const videoRef = useRef(null);

....

<TwilioVideo
        ref={videoRef}
	...

```

Then you use it using `videoRef.current`

**Review by fabiendem — COMMENTED** [review]
Added a minor suggestion

File: README.md (hunk 1 above)
Comment (markokarapandzic) [reply README.md:228]: Yeah, it's a better solution. I will commit new changes to it.

Comment (markokarapandzic) [reply README.md]: I added the useRef Hook. You can check it out in the last commit.

**Review by slycoder — APPROVED** [review]
Cool, like the modern approach to this.  Have you considered also updating the corresponding code in the Example/ directory?

**Comment by markokarapandzic** [comment]
I can check it out. And see what can be changed.

**Comment by slycoder** [comment]
Thank you so much!  I can also go ahead and merge this and we can tackle the example app in a separate PR if you prefer.

**Comment by markokarapandzic** [comment]
Ok, we can do it like that.

**Comment by markokarapandzic** [comment]
I checked out the code. So you want me to just change the code in the *index.js* from Class Component to the Function Components? Same as I did here.

**Comment by slycoder** [comment]
Yep that's exactly what I had in mind.  Many thanks @markokarapandzic  for taking this on!

**Comment by markokarapandzic** [comment]
Ok, you can merge this. And I will tackle *index.js* changes in another PR.

**Pull request merged**